packages/graph-engine/src/nodes/curves/sample.ts

```diff
@@ -51,15 +51,21 @@ export default class NodeDefinition extends Node {
     super(props);
     this.addInput("curve", {
       type: CurveSchema,
-      visible: true,
     });
     this.addInput("sample", {
-      type: NumberSchema,
-      visible: true,
+      type: {
+        ...NumberSchema,
+        default: 0.5,
+      }
     });
     this.addOutput("value", {
       type: Vec2Schema,
-      visible: true,
+    });
+    this.addOutput("x", {
```

Comment:
Note. We should probably remove the vec2 as it can be reconstructed from the two outputs

Reply:
lets leave it in for now, we might have different usecases soon with 3d gradient